o11y/tracer/datadog/datadog.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package datadog
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
+
+	"github.com/DataDog/chaos-controller/o11y/tracer/types"
+)
+
+// Sink describes a datadog tracer sink
+type Sink struct{}
+
+// New initiated datadog tracer sink
+func New(cfg types.SinkConfig) (Sink, error) {
+	var err error
+
+	if cfg.EnableTracer {
+		tracer.Start(
+			tracer.WithSampler(tracer.NewRateSampler(cfg.SampleRate)),
+		)
+	}
+
+	if cfg.EnableTracer && cfg.EnableProfiler {
```

Comment:
we can't have a profiler without a tracer?

Reply:
It can; I'll remove the condition.